packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -800,9 +821,12 @@ export const Dropdown = (React.forwardRef<HTMLDivElement, DropdownProps>((props,
       }),
     );
     return (
-      <div role="listbox" tabIndex={-1} aria-label={a11ySelectedItemsMessage}>
-        {selectedItems}
-      </div>
+      <>
+        <div role="listbox" tabIndex={-1} aria-label={a11ySelectedItemsMessage}>
+          {selectedItems}
+        </div>
+        {selectedItemsCountNarration}
```

Comment:
wouldn't be better to put the message above of listbox? when you navigate in virtual cursor to input first you will hit the message? 
or it is better to find the message after listbox?

Reply:
I think the position between the listbox and input field is better, because this way it is clear to which component the message belongs. If it were either above or belong the component, it might not be obvious when more such components ar next to each other.